tests/server_send.cpp

```diff
@@ -25,6 +25,8 @@
 using namespace ioremap::elliptics;
 using namespace boost::unit_test;
 
+#define ENABLE_LONG_TESTS 0
```

Comment:
It depends on internal timeout for write command used by server-send, currently this timeout is 60 seconds and can not be changed from this point.